packages/composite-checkout/src/lib/line-items.js

```diff
@@ -25,3 +25,13 @@ export function useTotal() {
 	const [ , total ] = useLineItems();
 	return total;
 }
+
+export function useHasDomainsInCart() {
```

Comment:
This is very clever. Nice custom hook! Since you've put this in the public api, can you add a brief description of it in the README in the API section (it's sorted alphabetically)?

Reply:
Sure thing. I'd be happy to.